docs/README.md

```diff
@@ -54,9 +54,10 @@ Executing these steps separately is equivalent to calling `predict_proba` on the
 <!--pytest-codeblocks:cont-->
 
 ```python
+# Predict probability for a single example
 y_proba_fhe = model.predict_proba(X_test[[0]], fhe="execute")
 
-# Quantize an input (float)
+# Quantize a original float input
```

Comment:
an*

Reply:
this appears to be your speciality, @kcelia